### DISCUSSION TITLE
Why would we use this over the default HomeKit integration?

### DISCUSSION THREAD

**Discussion by bwt615** [homebridge-plugins/homebridge-august#107] [Q&A]
I have latest WiFi lock, with a bridge. Two useful reasons I can see for this:

- Force HomeKit to use WiFi and the bridge to unlock versus Bluetooth from say a HomePod
- Save the need from an extra dummy switch to get around location based unlock, hence saving the app.

Am I thinking about this right? I couldn't find this in either questions or read me.

**Comment by donavanbecker** [comment]
Also some locks don't support native HomeKit

**Reply by bwt615** [reply]
Awesome! But is my understanding correct?

**Reply by donavanbecker** [reply]
> Awesome! But is my understanding correct?

Yes

**Reply by bwt615** [reply]
> > Awesome! But is my understanding correct?
> 
> Yes

So how do I control the device type then? I just tried this and it gave me the prompt to unlock automatically.
